jekyll/_cci2/using-macos.md

```diff
@@ -66,9 +66,9 @@ To read about CircleCI's customer support policy regarding beta images, please c
 ### Apple silicon support
 {: #apple-silicon-support }
 
-It is possible to build Apple Silicon/universal binaries using Xcode `12.0.0` and higher, as Apple provides both the Intel (`x86_64`) and Apple Silicon (`arm64`) toolchains in this release. Cross-compiling Apple Silicon binaries on Intel hosts has an additional overhead, and, as a result, compilation times will be longer than native compilation for Intel.
+Running or testing Apple Silicon apps natively can be done on our silicon-based Macs or by using [CircleCI runner]({{site.baseurl}}/runner-overview/#supported).
```

Comment:
> https://circleci.com/docs/runner-overview/#supported

We did a bit of a refresh on the runner content some time back and this path is no longer valid. Would you mind double checking the runner docs again and adding the URL to the section you want to include? Unfortunately it's not immediately evident to me just from reading the page again :\

Reply:
Updated! Thanks for pointing this out.